saved-to-the-uttermost/updated/06-chapter-5.adoc

```diff
@@ -445,14 +450,16 @@ what need would there be for any entreating them to be reconciled?
 Ambassadors are not sent after peace is already perfected and reconciliation made,
 but to entreat for reconciliation.
 
+[.discourse-part]
 Objection: Secondly, they object (fRom. 5:21 of the same chapter),
```

Comment:
just happened to notice this, though not on a line you touched at all.  my scripture reference finder made a big mistake here, it changed `(from v. 21 of the same...)` to `(fRom. 5:21 of the same...`. It wrongly thought that the "rom" of "from" was an abbreviation for Romans.  Can you switch that back while you're at it. I'm going to fix the scripture finder itself so this won't happen again.